Cartfile.private

```diff
@@ -1,2 +0,0 @@
-github "Quick/Quick"
```

Comment:
Are you sure we should remove these Carthage files?

Reply:
These libraries are not being used